shell/browser/font/electron_font_access_delegate.cc

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2021 Microsoft, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#include <utility>
+
+#include "shell/browser/font/electron_font_access_delegate.h"
+
+#include "base/task/post_task.h"
+#include "chrome/browser/browser_process.h"
+#include "content/public/browser/browser_task_traits.h"
+#include "content/public/browser/font_access_chooser.h"
+#include "content/public/browser/render_frame_host.h"
+
+class ElectronFontAccessChooser : public content::FontAccessChooser {
+ public:
+  explicit ElectronFontAccessChooser(base::OnceClosure close_callback) {
+    base::PostTask(
+        FROM_HERE, {content::BrowserThread::UI},
+        base::BindOnce(
+            [](base::OnceClosure closure) { std::move(closure).Run(); },
+            std::move(close_callback)));
```

Comment:
It seems like you could probably pass `std::move(close_callback)` here rather than re-wrapping it, yes? i.e.
```suggestion
    base::PostTask(FROM_HERE, {content::BrowserThread::UI}, std::move(close_callback));
```